packages/@sanity/imagetool/src/HotspotImage.js

```diff
@@ -59,7 +58,6 @@ export default class HotspotImage extends React.PureComponent {
         "Image '%s' already loaded, refreshing (from cache) to trigger onLoad / onError",
         this.props.src
       )
-      imageElement.src = imageElement.src
```

Comment:
Part of the linting changes, by removing this then it will simply use the props src during the render :)